source/includes/read/specify-queries.rb

```diff
@@ -0,0 +1,79 @@
+require 'bundler/inline'
+
+gemfile do
+  source 'https://rubygems.org'
+  gem 'mongo'
+end
+
+uri = '<connection string>'
+
+Mongo::Client.new(uri) do |client|
+    # start-setup
+    database = client.use('db')
+    collection = database[:fruits]
+
+    # Inserts documents representing fruits
+    fruits = [
+    { _id: 1, name: 'apples', qty: 5, rating: 3, color: 'red', type: ['fuji', 'honeycrisp'] },
+    { _id: 2, name: 'bananas', qty: 7, rating: 4, color: 'yellow', type: ['cavendish'] },
+    { _id: 3, name: 'oranges', qty: 6, rating: 2, type: ['naval', 'mandarin'] },
+    { _id: 4, name: 'pineapples', qty: 3, rating: 5, color: 'yellow' }
+    ]
+
+    collection.insert_many(fruits)
+    # end-setup
+
+    # Retrieves documents in which the "color" value is "yellow"
+    # start-find-exact
+    filter = { color: 'yellow' }
+    results = collection.find(filter)
+    results.each do |doc|
+      puts doc
+    end
+    # end-find-exact
```

Comment:
```suggestion
    # start-find-exact
    # Retrieves documents in which the "color" value is "yellow"
    filter = { color: 'yellow' }
    results = collection.find(filter)
    results.each do |doc|
      puts doc
    end
    # end-find-exact
```
S: I usually like to include the comments in the snippet, so users will have it when they copy the example. More of a style preference I suppose, but we should be consistent. Let me know what you think!

Reply:
I don't think we need to make these descriptive code comments visible to the user, since we already introduce each code example with a more detailed description.  But agreed that we should be consistent with that!